### PR TITLE
Clean the description file #82

### PR DESCRIPTION
**Pull request by Degoot-AM**
This PR aims at addressing issue #82

### CODE REVIEW

**Review by jamesmbaazam — CHANGES_REQUESTED** [review]
Hi @Degoot-AM, could you take this opportunity to do some extra cleanup? In particular:

- The Title field must be capitalized.
- In the Authors field, do well to recognise all contributions so far, i.e., reviewers, other submissions, and authors.

Also note that since {numberize} is not on CRAN, you may have to find a way not to Import it unless it'll go on CRAN.

**Comment by Degoot-AM** [comment]
Thank you @jamesmbaazam for your comments. I have linked some issues about the comments that are out of scope of this PR. Will be addressed in separate PRs.
> Hi @Degoot-AM, could you take this opportunity to do some extra cleanup? In particular: 
> * The Title field must be capitalized.

> * In the Authors field, do well to recognise all contributions so far, i.e., reviewers, other submissions, and authors.

       This will be address by issue #88. 

> * Also note that since {numberize} is not on CRAN, you may have to find a way not to Import it unless it'll go on CRAN.

    This will be addressed by #89. 

> * See [this chapter in the R Packages book](https://r-pkgs.org/description.html) for more detail on how the DESCRIPTION should be standardized.

File: DESCRIPTION [hunk 1]
```diff
@@ -15,9 +15,11 @@ Authors@R: c(
            comment = c(ORCID = "0000-0002-4094-1476"))
   )
 Maintainer: Karim Mané <karim.mane@lshtm.ac.uk>
-Description: cleanepi provides functions for cleaning and standardizing tabular data, tailored specifically for curating epidemiological data.
+Description: cleanepi provides functions for cleaning and standardizing
```
Comment (jamesmbaazam) [inline DESCRIPTION:18]:
```suggestion
Description: Provides functions for cleaning and standardizing
```

**Review by jamesmbaazam — APPROVED** [review]
Thanks for making the change, Degoot. I've made one last suggestion, which you can just commit from here.

**Pull request merged**